front-end-components/src/views/compare-your-workforce/partials/headcount.tsx

```diff
@@ -56,7 +57,9 @@ export const Headcount: React.FC<HeadcountProps> = (props) => {
       >
         <h2 className="govuk-heading-m">School workforce (Headcount)</h2>
         <ChartDimensions
-          dimensions={WorkforceCategories}
+          dimensions={WorkforceCategories.filter(
+            (category) => category !== HeadcountPerFTE
```

Comment:
Need to remove "PercentageOfWorkforce" from the selection too.

Reply:
pushed with this removed